#!/usr/bin/env python

PACKAGE = "kinect_tomato_searcher"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_min", int_t, 0, "color min(less then)",  10, 0, 360)
gen.add("h_max", int_t, 0, "color max(more then)", 300, 0, 360)
gen.add("s_min", int_t, 0, "saturation min",        30, 0, 100)
gen.add("s_max", int_t, 0, "saturation max",       100, 0, 100)
gen.add("v_min", int_t, 0, "saturation min",        50, 0, 100)
gen.add("v_max", int_t, 0, "saturation max",       100, 0, 100)

exit(gen.generate(PACKAGE, "kinect_tomato_searcher", "Search"))
